### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by priyanka-surana**
The PR updates the main `index.php` document.

### CODE REVIEW

File: public_html/index.php [hunk 1]
```diff
@@ -348,9 +348,9 @@
       <div class="card h-100 shadow-sm">
         <div class="card-body d-flex align-items-center">
           <div>
-            <h5 class="card-title">Cloud-ready</h5>
-            <p class="card-text">Pipelines are tested on AWS after every release.
-              You can even browse results live on the website and use outputs for your own benchmarking.
+            <h5 class="card-title">Robust</h5>
```
Comment (gq1) [inline public_html/index.php:351]: Why remove `Cloud-ready`?

File: public_html/index.php [hunk 2]
```diff
@@ -348,9 +348,9 @@
...
+            <p class="card-text">Pipelines are tested on data from different parts of the tree of life.
```
Comment (gq1) [inline public_html/index.php:352]: `the tree of life` or just `Tree of Life`?

Comment (gq1) [reply public_html/index.php:352]: We can mention viewing the results on Tower and S3 later.

File: public_html/index.php (hunk 1 above)
Comment (muffato) [reply public_html/index.php:351]: I would keep "Cloud-ready" as we use containers for everything

Comment (priyanka-surana) [reply public_html/index.php:351]: But we do not test the results on cloud. We think it will work on the cloud but we do not have tests to show for it. And at this time we cannot browse our results externally, at least based on what I know.

File: public_html/index.php (hunk 2 above)
Comment (priyanka-surana) [reply public_html/index.php:352]: This tree of life is the taxonomy not our programme.

File: public_html/index.php [hunk 3]
```diff
@@ -259,11 +259,11 @@
     </div>
     <div>
       <p class="d-flex position-relative justify-content-center text-white m-0 pb-3">
-        <img src="/assets/img/nature_biotech.svg" alt="Nature Biotechnology" width="60px" class="float-start me-3">
-        <a class="text-white stretched-link text-decoration-none d-flex flex-column justify-content-center" href="https://doi.org/10.1038/s41587-020-0439-x" target="_blank">
-          nf-core is published in Nature Biotechnology!
-          <span class="underline m-auto"><em>Nat Biotechnol</em> <strong>38</strong>, 276–278 (2020).
-            <img src="/assets/img/OpenAccess.svg" alt="Nature Biotechnology" height="20px" class="ms-2"></span>
+        <img src="/assets/img/pnas.png" alt="PNAS" width="60px" class="float-start me-3">
```
Comment (gq1) [inline public_html/index.php:262]: the name should be `pnas-logo.png`

File: public_html/index.php (hunk 2 above)
Comment (priyanka-surana) [reply public_html/index.php:352]: Yes Tower and S3 are a bit glitchy at the moment. I thought we could mention it once we figure out how to display the results for the pipelines.

File: public_html/index.php (hunk 3 above)
Comment (priyanka-surana) [reply public_html/index.php:262]: I will update. Thank you for catching that.

File: public_html/index.php (hunk 1 above)
Comment (gq1) [reply public_html/index.php:351]: All NF pipelines are supposed to work on the cloud? We can treat our farm as cloud.

Comment (muffato) [reply public_html/index.php:351]: Some pipelines are properly tested on GitHub CI, which amounts to a cloud. There are two issues, I think:

- I think it's only TreeVal that's currently incompatible because it needs a lot of data from Lustre.
- We hit some problems with Busco, but I think it's essentially because we wanted to force running it on a specific test dataset. We could just skip the parameter and the pipeline should happily run, downloading the dataset live from the Busco website

**Comment by priyanka-surana** [comment]
@gq1 The other changes are showing on the website but not the colour update. Did I not change all the variables for that?

**Comment by priyanka-surana** [comment]
> @gq1 The other changes are showing on the website but not the colour update. Did I not change all the variables for that?

Now it is.

**Comment by priyanka-surana** [comment]
For the first blue section, where we have the text `Highly optimised pipelines with excellent reporting. Validated releases ensure reproducibility.` The heading `For facilities` does not show up in dark mode. How to fix that?

File: public_html/index.php (hunk 1 above)
Comment (priyanka-surana) [reply public_html/index.php:351]: If we are keeping `cloud-ready`, I will not change the logo. Otherwise I need to change the `aws` logo too.

Comment (priyanka-surana) [reply public_html/index.php:351]: I feel one of the features of "cloud ready" is the ability of pipelines to create instances as needed within the given parameters while talking to all the components successfully. We have not tested any pipeline on a full dataset on the cloud (even GitHub). 

But if there is general consensus, I can put it back in.

File: public_html/index.php [hunk 4]
```diff
@@ -384,7 +384,7 @@
           <div class="card-body">
             <h5 class="card-title">Start from the template</h5>
             <div class='d-flex align-items-center'>
-              <p class="card-text">All pipelines and modules must be based on nf-core templates and be created using <code>nf-core</code> <a href="https://github.com/nf-core/tools">tools</a>.</p>
+              <p class="card-text">All pipelines and modules must be based on nf-core templates and be created using <a href="https://github.com/nf-core/tools">nf-core tools</a>.</p>
```
Comment (gq1) [inline public_html/index.php:387]: is this change necessary?

File: public_html/index.php (hunk 1 above)
Comment (priyanka-surana) [reply public_html/index.php:351]: I am adding back cloud ready. Should we change the image to a cloud or leave it as `aws`, seems like free advertising.

Comment (gq1) [reply public_html/index.php:351]: That just represents the cloud, make less change as possible.

File: public_html/index.php (hunk 4 above)
Comment (priyanka-surana) [reply public_html/index.php:387]: It does not look good, without the change. Lets implement it.

**Comment by priyanka-surana** [comment]
> For the first blue section, where we have the text `Highly optimised pipelines with excellent reporting. Validated releases ensure reproducibility.` The heading `For facilities` does not show up in dark mode. How to fix that?

The change has been made, can we pull it through and see how it looks? https://github.com/nf-core/nf-co.re/pull/1641

File: public_html/index.php [hunk 5]
```diff
@@ -272,8 +272,8 @@
 
 
 <div class="container py-5 text-center lead" id="features">
-  <p>Nextflow is an incredibly powerful and flexible workflow language.</p>
-  <p><strong>nf-core</strong> pipelines adhere to strict guidelines - if one works, they all will.</p>
+  <p>Tree of Life production suite is designed to handle nature's diversity.</p>
```
Comment (muffato) [inline public_html/index.php:275]: Missing "The" ?

File: public_html/index.php [hunk 6]
```diff
@@ -308,7 +308,7 @@
         <div class="card-body d-flex align-items-center">
           <div>
             <h5 class="card-title">Stable Releases</h5>
-            <p class="card-text">nf-core pipelines use GitHub releases to tag stable versions of the code
+            <p class="card-text">sanger-tol pipelines use GitHub releases to tag stable versions of the code
               and software, making pipeline runs totally reproducible.</p>
```
Comment (muffato) [inline public_html/index.php:312]: Can we mention Zenodo here ?

File: public_html/index.php [hunk 7]
```diff
@@ -336,8 +336,8 @@
             <p class="card-text">
               Pipelines follow best-practices to ensure maximum
               portability and reproducibility.
-              The large community makes the pipelines exceptionally
-              well tested and easy to run.
+              Scale and diversity in Tree of Life makes the pipelines
```
Comment (muffato) [inline public_html/index.php:339]:
```suggestion
              Scale and diversity in Tree of Life make the pipelines 
```

**Review by muffato — COMMENTED** [review]
Where did you find the PNAS logo ? The version on Wikipedia https://commons.wikimedia.org/wiki/File:PNAS-logo-primary-2c.png is under CC Attribution-Share Alike, so we need to give credit

File: public_html/index.php (hunk 1 above)
Comment (priyanka-surana) [reply public_html/index.php]: Fixed

File: public_html/index.php (hunk 5 above)
Comment (priyanka-surana) [reply public_html/index.php]: Done

File: public_html/index.php (hunk 6 above)
Comment (priyanka-surana) [reply public_html/index.php]: Done
